app/assets/stylesheets/_forms.scss

```diff
@@ -0,0 +1,42 @@
+@import "bootstrap";
```

Comment:
have imported bootstrap in `app/assets/stylesheets/custom.scss`, have to do this again for every file?